client/packages/invoices/src/InboundShipment/DetailView/DetailView.tsx

```diff
@@ -66,9 +73,10 @@ const DetailViewInner = () => {
     mode: returnModalMode,
     setMode,
   } = useEditModal<string[]>();
-  const { info } = useNotification();
-  const { clearSelected } = useTableStore();
+
+  const { data, isLoading, invalidateQuery } = useInbound.document.get();
   const { data: vvmStatuses } = useVvmStatusesEnabled();
+  const isDisabled = useInbound.utils.isDisabled();
```

Comment:
sorry just rearranging here, I like having the helper hooks and the data hooks separate